packages/x-tree-view/src/internals/plugins/useTreeViewItems/useTreeViewItems.types.ts

```diff
@@ -41,7 +41,13 @@ export interface UseTreeViewItemsPublicAPI<R extends {}> {
    */
   getItemTree: () => TreeViewBaseItem[];
   /**
-   * Get the id of the parent item.
+   * Toggle the disabled state of the item with the given id.
+   * @param {object} params The params of the method.
+   * @param {TreeViewItemId } params.itemId The id of the item to get the children of.
+   * @param {boolean } params.shouldBeDisabled true if the item should be disabled.
+   */
+  setIsItemDisabled: (params: { itemId: TreeViewItemId; shouldBeDisabled?: boolean }) => void;
```

Comment:
```suggestion
  setIsItemDisabled: (parameters: { itemId: TreeViewItemId; shouldBeDisabled?: boolean }) => void;
```

(same in JSDoc)

Nitpick, I did the same in #16795, I think it's closer from the new nomenclature from the core teams.